packages/valory/skills/transaction_settlement_abci/test_tools/integration.py

```diff
@@ -142,7 +142,24 @@ def sign_tx(self) -> None:
             participant_to_signature[address] = SignaturePayload(
                 sender=address,
                 signature=signature_hex,
-            )
+            ).json
+
+        # FIXME: The following loop is a patch. The
```

Comment:
Please open an issue for this and reference here and on Trello.

Reply:
Opened #1779 and https://trello.com/c/PxjTZMJz.